src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,37 @@
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder builder = new StringBuilder("Report for period ")
+                                                    .append(dateFrom).append(" - ")
+                                                    .append(dateTo);
+        int[] salary = new int[names.length];
+        for (int i = 0; i < salary.length; i++) {
+            for (String info : data) {
```

Comment:
```suggestion
        for (int i = 0; i < salary.length; i++) {
            int salary = 0;
            for (String info : data) {
```
you don't need an array